### ISSUE TITLE
any progress on training code?

### ISSUE THREAD

**Issue by mdering** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#1]
Have you made a faithful interpretation of their generation code for training?

**Comment by tensorboy** [comment]
Hi, @mdering. Thanks for your attention for that reproduced code. I'm new for pose estimation and still studying how Part Affinity Fields works.

**Comment by mdering** [comment]
that's too bad. the source material is somewhat ambiguous on their choices of thresholds and which space they're calculated in. Let me know if you find anything implemented in python

**Comment by tensorboy** [comment]
Yes, I almost got crazy for their source code in [here](https://github.com/CMU-Perceptual-Computing-Lab/caffe_train/blob/master/src/caffe/cpm_data_transformer.cpp) and the parameters are [here](https://github.com/CMU-Perceptual-Computing-Lab/caffe_train/blob/10275384a3980ffa6a99061d01e7256b31a68c43/src/caffe/proto/caffe.proto) (In caffe.proto file. message CPMTransformationParameter defined all the parameters for data augmentation).

Thank you @mdering !

**Comment by mdering** [comment]
the code and parameters i get, but the augmentation stuff, and the masking procedures are very hard to parse in c++.

**Comment by ouceduxzk** [comment]
I am also working on rewriting this code, for me the difficult part is the rtpose.cpp where the PAF calculation occurs, I get the big picture but many details remain to be clear . I think the cpm_transformation should be easier though might also takes some time.

**Comment by tensorboy** [comment]
@ouceduxzk  happy to know you also trying to reproduce it!

**Comment by ouceduxzk** [comment]
after trying with this code for a while, I think I understand what each part do. I would like to implement it in mxnet later on

**Comment by tensorboy** [comment]
Sounds great!

**Comment by mdering** [comment]
an update to this, I still have not found a model that even comes close to converging during training. I am truly stumped.

**Comment by tensorboy** [comment]
Try `train.py`
